cli/azd/pkg/common/detailed_error.go

```diff
@@ -0,0 +1,28 @@
+package common
```

Comment:
I realized that we did not close on this, but during the discussion in https://github.com/Azure/azure-dev-docs/issues/6, I felt that we were leaning towards avoiding introducing `core` or `common` packages (.NET idiom), but if it's truly a "core helper" to just include it under `internal/` package (go idiom).